### ISSUE TITLE
Release 4.9.0 - Beta 1 - Wazuh UI regression testing

### ISSUE THREAD

**Issue by asteriscos** [wazuh/wazuh-dashboard-plugins#6897]
The following issue aims to run manual tests for the current release candidate, report the results, and open new issues for any encountered errors.

## Wazuh UI tests information
|                                        |     |
|----------------------------------------|-----|
| **Main release candidate issue**       |  https://github.com/wazuh/wazuh/issues/25025   |
| **Version**                            |   4.9.0  |
| **Release candidate #**                |  Beta 1  |
| **Tag**                                |  [v4.9.0-beta1](https://github.com/wazuh/wazuh-dashboard-plugins/releases/tag/v4.9.0-beta1)   |
| **Previous UI regression tests issue** |   https://github.com/wazuh/wazuh-dashboard-plugins/issues/6860  |

## Test report procedure

**TL;DR**
   1. The specified tests will be executed in every platform and version mentioned in this issue.
   1. Include evidence of each test performed.
   1. Report any problem or bug. Open a new issue for each of them and link them here.
   1. Justify skipped tests.

All test results must have one the following statuses: 
|                 |                                                                      |
|-----------------|----------------------------------------------------------------------|
| :black_circle:  | The test hasn't started yet.                                         |
| :green_circle:  | All checks passed.                                                   |
| :red_circle:    | There is at least one failed check.                                  |
| :yellow_circle: | There is at least one expected fail or skipped test and no failures. |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause.
It must be included in the `Problems` section of the current release candidate issue.

Any expected fail or skipped test must be justified with a reason. 
All auditors must validate the justification for an expected fail or skipped test.

An extended report of the test results must be attached as a zip, txt or images. 
This report can be used by the auditors to dig deeper into any possible failures and details.

## Test

| Test                                                                 | Chrome         | Firefox        | Safari         |
|---------------------------------------------------------------------|----------------|----------------|----------------|
| Verify the app package installs and operates as expected | :green_circle:   | :green_circle: | :white_circle: |
| Fix endpoints summary agent action buttons navigation | :green_circle:  | :green_circle: | :white_circle: |
| Added consistency between url and Endpoint groups view | :green_circle: | :green_circle: | :white_circle: |
| Fixes redirects to specific techniques | :green_circle: | :green_circle: | :white_circle: |
| Add tooltips to new discover table call to actions | :green_circle: | :green_circle: | :white_circle: |
| Fixed render bug in security alerts table | :green_circle: | :green_circle: | :white_circle: |
| Display rule details flyout based on URL | :green_circle: | :green_circle: | :white_circle: |
| Leave the date of last modification on a single line on the FIM > Inventory > Windows | :green_circle: | :green_circle: | :white_circle: |
| The dashboard of the mitre module is the same with or without a pinned agent | :green_circle: | :green_circle: | :white_circle: |
| Checking the internal tab when changing agent on FIM > Inventory > Windows registry | :green_circle: | :green_circle: | :white_circle: |
| Fixes report generation when have a pinned agent | :green_circle: | :red_circle: | :white_circle: |
| Add field links in discover document details | :green_circle: | :green_circle: | :white_circle: |
| Add view single document and surrounding documents | :green_circle: | :green_circle: | :white_circle: |
| Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background | :green_circle: | :green_circle: | :white_circle: |
| Clear session button text is displayed in full | :green_circle: | :green_circle: | :white_circle: |
| Unification the rendering of flyout columns | :green_circle: | :green_circle: | :white_circle: |
| Fix maximized visualizations styles | :green_circle: | :green_circle: | :white_circle: |
| The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module | :green_circle: | :green_circle: | :white_circle: |
| Remove rule.mitre_techniques column from the Threat Hunting | :green_circle: | :green_circle: | :white_circle: |
| Some links in the threat hunting dashboard table do not redirect or have no links | :green_circle: | :green_circle: | :white_circle: |
| Add colours to GitHub dashboard metric visualisations | :green_circle: | :green_circle: | :white_circle: |
| Fixed donut card legend | :green_circle: | :green_circle: | :white_circle: |
| Fix permalink field in Events tab in Virustotal | :green_circle: | :green_circle: | :white_circle: |

## Test plan

1. Verify the app package installs and operates as expected.
2. [Fix endpoints summary agent action buttons navigation](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6893)
3. [Added consistency between url and Endpoint groups view](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6890)
4. [Fixes redirects to specific techniques](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6882)
5. [Add tooltips to new discover table call to actions](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6889)
6. [Fixed render bug in security alerts table](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6883)
7. [Display rule details flyout based on URL](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6886)
8. [Leave the date of last modification on a single line on the FIM > Inventory > Windows](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6876)
9. [The dashboard of the mitre module is the same with or without a pinned agent](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6878)
10. [Checking the internal tab when changing agent on FIM > Inventory > Windows registry](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6880)
11. [Fixes report generation when have a pinned agent](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6873)
12. [Add field links in discover document details](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6843)
13. [Add view single document and surrounding documents](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6831)
14. [Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6868)
15. [Clear session button text is displayed in full](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6871)
16. [Unification the rendering of flyout columns](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6848)
17. [Fix maximized visualizations styles](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6865)
18. [The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6847)
19. [Remove rule.mitre_techniques column from the Threat Hunting](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6858)
20. [Some links in the threat hunting dashboard table do not redirect or have no links](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6845)
21. [Add colours to GitHub dashboard metric visualisations](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6857)
22. [Fixed donut card legend](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6832)
23. [Fix permalink field in Events tab in Virustotal](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6839)

## Conclusions :red_circle:

Most of the tests were performed successfully, but I found an issue when trying to generate a PDF report in Firefox:
- https://github.com/wazuh/wazuh-dashboard-plugins/issues/6900

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @asteriscos

**Comment by asteriscos** [comment]
| Test                                                                 | Chrome         | Firefox        | Safari         |
|---------------------------------------------------------------------|----------------|----------------|----------------|
| Verify the app package installs and operates as expected | :black_circle:   | :black_circle: | :black_circle: |
| Fix endpoints summary agent action buttons navigation | :black_circle:  | :black_circle: | :black_circle: |
| Added consistency between url and Endpoint groups view | :black_circle: | :black_circle: | :black_circle: |
| Fixes redirects to specific techniques | :black_circle: | :black_circle: | :black_circle: |
| Add tooltips to new discover table call to actions | :black_circle: | :black_circle: | :black_circle: |
| Fixed render bug in security alerts table | :black_circle: | :black_circle: | :black_circle: |
| Display rule details flyout based on URL | :black_circle: | :black_circle: | :black_circle: |
| Leave the date of last modification on a single line on the FIM > Inventory > Windows | :black_circle: | :black_circle: | :black_circle: |
| The dashboard of the mitre module is the same with or without a pinned agent | :black_circle: | :black_circle: | :black_circle: |
| Checking the internal tab when changing agent on FIM > Inventory > Windows registry | :black_circle: | :black_circle: | :black_circle: |
| Fixes report generation when have a pinned agent | :black_circle: | :black_circle: | :black_circle: |
| Add field links in discover document details | :black_circle: | :black_circle: | :black_circle: |
| Add view single document and surrounding documents | :black_circle: | :black_circle: | :black_circle: |
| Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background | :black_circle: | :black_circle: | :black_circle: |
| Clear session button text is displayed in full | :black_circle: | :black_circle: | :black_circle: |
| Unification the rendering of flyout columns | :black_circle: | :black_circle: | :black_circle: |
| Fix maximized visualizations styles | :black_circle: | :black_circle: | :black_circle: |
| The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module | :black_circle: | :black_circle: | :black_circle: |
| Remove rule.mitre_techniques column from the Threat Hunting | :black_circle: | :black_circle: | :black_circle: |
| Some links in the threat hunting dashboard table do not redirect or have no links | :black_circle: | :black_circle: | :black_circle: |
| Add colours to GitHub dashboard metric visualisations | :black_circle: | :black_circle: | :black_circle: |
| Fixed donut card legend | :black_circle: | :black_circle: | :black_circle: |
| Fix permalink field in Events tab in Virustotal | :black_circle: | :black_circle: | :black_circle: |

<details>

<summary>Verify the app package installs and operates as expected :black_circle: </summary>



</details> 

<details>

<summary>Fix endpoints summary agent action buttons navigation :black_circle: </summary>



</details> 

<details> 
<summary>Added consistency between url and Endpoint groups view :black_circle: </summary>



</details> 

<details> 
<summary>Fixes redirects to specific techniques :black_circle: </summary>



</details> 

<details> 
<summary>Add tooltips to new discover table call to actions :black_circle: </summary>



</details> 

<details> 
<summary>Fixed render bug in security alerts table :black_circle: </summary>



</details> 

<details> 
<summary>Display rule details flyout based on URL :black_circle: </summary>



</details> 

<details> 
<summary>Leave the date of last modification on a single line on the FIM > Inventory > Windows :black_circle: </summary>



</details> 

<details> 
<summary>The dashboard of the mitre module is the same with or without a pinned agent :black_circle: </summary>



</details> 

<details> 
<summary>Checking the internal tab when changing agent on FIM > Inventory > Windows registry :black_circle: </summary>



</details> 

<details> 
<summary>Fixes report generation when have a pinned agent :black_circle: </summary>



</details> 

<details> 
<summary>Add field links in discover document details :black_circle: </summary>



</details> 

<details> 
<summary>Add view single document and surrounding documents :black_circle: </summary>



</details> 

<details> 
<summary>Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background :black_circle: </summary>



</details>

<details>  
<summary>Clear session button text is displayed in full :black_circle: </summary>



</details> 

<details>  
<summary>Unification the rendering of flyout columns :black_circle: </summary>



</details> 

<details>  
<summary>Fix maximized visualizations styles :black_circle: </summary>



</details> 

<details>  
<summary>The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module :black_circle: </summary>



</details> 

<details>  
<summary>Remove rule.mitre_techniques column from the Threat Hunting :black_circle: </summary>



</details> 

<details>  
<summary>Some links in the threat hunting dashboard table do not redirect or have no links :black_circle: </summary>



</details> 
<details>  
<summary>Add colours to GitHub dashboard metric visualisations :black_circle: </summary>



</details> 
<details>  
<summary>Fixed donut card legend :black_circle: </summary>



</details> 

<details>  
<summary>Fix permalink field in Events tab in Virustotal :black_circle: </summary>



</details>

**Comment by chantal-kelm** [comment]
| Test                                                                 | Chrome         | Firefox        | Safari         |
|---------------------------------------------------------------------|----------------|----------------|----------------|
| Verify the app package installs and operates as expected | :green_circle:   | :black_circle: | :black_circle: |
| Fix endpoints summary agent action buttons navigation | :black_circle:  | :black_circle: | :black_circle: |
| Added consistency between url and Endpoint groups view | :black_circle: | :black_circle: | :black_circle: |
| Fixes redirects to specific techniques | :black_circle: | :black_circle: | :black_circle: |
| Add tooltips to new discover table call to actions | :black_circle: | :black_circle: | :black_circle: |
| Fixed render bug in security alerts table | :black_circle: | :black_circle: | :black_circle: |
| Display rule details flyout based on URL | :black_circle: | :black_circle: | :black_circle: |
| Leave the date of last modification on a single line on the FIM > Inventory > Windows | :black_circle: | :black_circle: | :black_circle: |
| The dashboard of the mitre module is the same with or without a pinned agent | :black_circle: | :black_circle: | :black_circle: |
| Checking the internal tab when changing agent on FIM > Inventory > Windows registry | :black_circle: | :black_circle: | :black_circle: |
| Fixes report generation when have a pinned agent | :black_circle: | :black_circle: | :black_circle: |
| Add field links in discover document details | :black_circle: | :black_circle: | :black_circle: |
| Add view single document and surrounding documents | :black_circle: | :black_circle: | :black_circle: |
| Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background | :black_circle: | :black_circle: | :black_circle: |
| Clear session button text is displayed in full | :black_circle: | :black_circle: | :black_circle: |
| Unification the rendering of flyout columns | :black_circle: | :black_circle: | :black_circle: |
| Fix maximized visualizations styles | :black_circle: | :black_circle: | :black_circle: |
| The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module | :black_circle: | :black_circle: | :black_circle: |
| Remove rule.mitre_techniques column from the Threat Hunting | :black_circle: | :black_circle: | :black_circle: |
| Some links in the threat hunting dashboard table do not redirect or have no links | :black_circle: | :black_circle: | :black_circle: |
| Add colours to GitHub dashboard metric visualisations | :black_circle: | :black_circle: | :black_circle: |
| Fixed donut card legend | :black_circle: | :black_circle: | :black_circle: |
| Fix permalink field in Events tab in Virustotal | :black_circle: | :black_circle: | :black_circle: |

<details>

<summary>Verify the app package installs and operates as expected :green_circle: </summary>

<img width="1792" alt="Captura de pantalla 2024-08-06 a la(s) 9 39 55 a  m" src="https://github.com/user-attachments/assets/b0bffc6c-5ab7-46d1-ae03-7a5243588a34">

</details> 

<details>

<summary>Fix endpoints summary agent action buttons navigation :black_circle: </summary>



</details> 

<details> 
<summary>Added consistency between url and Endpoint groups view :black_circle: </summary>



</details> 

<details> 
<summary>Fixes redirects to specific techniques :black_circle: </summary>



</details> 

<details> 
<summary>Add tooltips to new discover table call to actions :black_circle: </summary>



</details> 

<details> 
<summary>Fixed render bug in security alerts table :black_circle: </summary>



</details> 

<details> 
<summary>Display rule details flyout based on URL :black_circle: </summary>



</details> 

<details> 
<summary>Leave the date of last modification on a single line on the FIM > Inventory > Windows :black_circle: </summary>



</details> 

<details> 
<summary>The dashboard of the mitre module is the same with or without a pinned agent :black_circle: </summary>



</details> 

<details> 
<summary>Checking the internal tab when changing agent on FIM > Inventory > Windows registry :black_circle: </summary>



</details> 

<details> 
<summary>Fixes report generation when have a pinned agent :black_circle: </summary>



</details> 

<details> 
<summary>Add field links in discover document details :black_circle: </summary>



</details> 

<details> 
<summary>Add view single document and surrounding documents :black_circle: </summary>



</details> 

<details> 
<summary>Fix flyout of the inventory tab of the FIM module the bottom of the dropdown shows a different grey than the background :black_circle: </summary>



</details>

<details>  
<summary>Clear session button text is displayed in full :black_circle: </summary>



</details> 

<details>  
<summary>Unification the rendering of flyout columns :black_circle: </summary>



</details> 

<details>  
<summary>Fix maximized visualizations styles :black_circle: </summary>



</details> 

<details>  
<summary>The selected filter is not applied when redirecting to the events tab in the file integrity monitoring module :black_circle: </summary>



</details> 

<details>  
<summary>Remove rule.mitre_techniques column from the Threat Hunting :black_circle: </summary>



</details> 

<details>  
<summary>Some links in the threat hunting dashboard table do not redirect or have no links :black_circle: </summary>



</details> 
<details>  
<summary>Add colours to GitHub dashboard metric visualisations :black_circle: </summary>



</details> 
<details>  
<summary>Fixed donut card legend :black_circle: </summary>



</details> 

<details>  
<summary>Fix permalink field in Events tab in Virustotal :black_circle: </summary>



</details>

**Comment by asteriscos** [comment]
The tests performed look good to me.
